python/ray/data/tests/test_csv.py

```diff
@@ -770,26 +747,16 @@ def test_csv_read_filter_non_csv_file(ray_start_regular_shared, tmp_path):
     # Single non-CSV file.
     error_message = "Failed to read CSV file"
     with pytest.raises(ValueError, match=error_message):
-        ray.data.read_csv(path3).schema()
-
-    # Single non-CSV file with filter.
-    error_message = "No input files found to read"
-    with pytest.raises(ValueError, match=error_message):
-        ray.data.read_csv(path3, file_extensions=["csv"]).schema()
```

Comment:
We shouldn't test that this raises a value error, because returning an empty dataset is also valid